Add Gentoo logo to admin pages

Patch by Gordon Bos

--- a/bubba-frontend/admin/views/default/_css/admin.css
+++ b/bubba-frontend/admin/views/default/_css/admin.css
@@ -232,6 +232,12 @@

 }

+div#gentoo-logo {
+	position : absolute;
+	top : 23px;
+	left : 425px;
+}
+
 button:hover {
   outline: none;
 }
--- a/bubba-frontend/admin/views/default/main_view.php
+++ b/bubba-frontend/admin/views/default/main_view.php
@@ -292,6 +292,9 @@
 					<a href="http://www.excito.com" target="_tab" class="ui-excito-link"><img id="ex_logo" src="<?=FORMPREFIX.'/views/'.THEME?>/_img/purple-X.png" alt="<?=_("Excito")?>" /></a>
                 			<span id="ui-main-byExcito">by Excito Sweden</span>
                 		</div>
+                		<div id="gentoo-logo">
+					<img id="img_gentoo" src="<?=FORMPREFIX.'/views/'.THEME?>/_img/gentoo_logo.png" alt="<?=_("Gentoo Linux")?>" />
+                		</div>
 		                <a href="#" id="a_logo" onclick="location.href='<?=FORMPREFIX?>'">
 				<img id="img_logo" src="<?=FORMPREFIX.'/views/'.THEME?>/_img/B3_logo.png" alt="<?=_("B3 start page")?>" title="<?=_("B3 start page")?>" />
                 		</a>
